_pages/clearance-process/rules.md

```diff
@@ -0,0 +1,56 @@
+- name: Publish Proposed Rule in Federal Register and submit request to OMB
+- name: OMB reviews proposed collection
+- name: Consider the public comments on the proposed rule
+- name: Publishes Final Rule and submits final request to OMB
+
+## Page Content:
+
+steps:
+- content: "[Agency develops the information request in tandem with the proposed rule.](#develop-the-information-collection-request-within-your-agency) Your agency may have an internal approval process at this stage"
+  agency: true
+  public: false
+  omb: false
+- content: "[Agency publishes proposed rule in the Federal Register](#Publish-Proposed-Rule-in-Federal-Register-and-submit-request-to-OMB) and submits a PRA request to OMB"
+  agency: true
+  public: true
+  omb: true
+- content: "[OMB reviews proposed collection](#OMB-reviews-proposed-collection), and any comments received"
+  agency: false
+  public: false
+  omb: true
+- content: "[Agency considers the public comments on the proposed rule](#Consider-the-public-comments-on-the-proposed-rule) and the information collection"
+  agency: true
+  public: false
+  omb: false
+- content: "[Publishes Final Rule and submits final request to OMB](#Publishes-Final-Rule-and-submits-final-request-to-OMB)"
+  agency: false
+  public: false
+  omb: true
+---
+
+For information collections associated with proposed rules there is a slightly different process that an agency must go through to get OMB approval.
+
+## Process overview for information collections associated with proposed rules 
+
+{% include process.html steps=page.steps %}
+
+### Develop the information collection request within your agency
+Agency develops the information request. Your agency may have an internal approval process at this stage.
+
+### Publish Proposed Rule in Federal Register and submit request to OMB
+When the notice of proposed rulemaking goes out for comment, an agency must state the following in the rule preamble: 1) that the proposed rule contains an information collection; 2) describe and identify the collection; and 3) indicate that it has been submitted to OMB for review.  This step and the solicitation for comments on the collection in the proposed rule replace the 60-day and 30-day Federal Register notices required in the more typical information collection process.  The agency must also concurrently submit the proposed collection to OMB for review.
+
+### OMB reviews proposed collection
+During OMB review, OMB will review the information collection and any comments received. At this stage, OMB may have comments on the collection and defer final review until the agency finalizes the associated rule. Alternately, OMB may indicate the collection is fine as is (preapproved).  Either action will conclude review of the proposed collection submitted to OMB. It is important to note that **neither of these actions grant final approval to collect the information, which will occur later in Step 5.
```

Comment:
@mmcmanusOMB Did you mean to make this last sentence bold? Or are those asterisks referring to footnotes. Assuming the former. If so, it would need a closing `**` at the end of the sentence to make it bold.
```
It is important to note that **neither of these actions grant final approval to collect the information, which will occur later in Step 5.
```
I _think_ that might be what is causing the CircleCi to raise a flag...

Reply:
Hmmm - I added the closing ** and still doesn't work.